.travis.yml

```diff
@@ -1,7 +1,9 @@
 ---
 language: node_js
 node_js:
+  - "8.0.0"
   - "8"
+  - "lts/*"
```

Comment:
`lts/*` and `8` are often going to be redundant; however, this config will allow us to quickly pick up the latest LTS version as soon as it is available.